### ISSUE TITLE
Support multiple inboxes

### ISSUE THREAD

**Issue by gfpacheco** [joeyespo/inbox-by-gmail-checker#3]
Here are my suggestions on how it could be done:
- By enabling a second button on the extensions bar (not sure if it's possible) (better if different colors to choose).
- By showing a second counter (and show a dropdown to ask which account to open on click).

**Comment by joeyespo** [comment]
> By enabling a second button on the extensions bar

Yeah, afaik this can't be done unless you manually install a new copy of the extension.

> By showing a second counter (and show a dropdown to ask which account to open on click)

This could work, although it'd get crowded fast. It's still limited in the number of accounts you can use at once. Also, I personally prefer not to have a dropdown since I think it takes away from the experience. (Of course, I don't have multiple inboxes, so I'm a little removed from this suggestion.)

What do you think about introducing a "priority"? Say, if you connect a secondary account, it would show its unread count in a different color, but only when the primary account was zero? This would keep the interface consistent. Although you'd lose out on some information.

Another idea is to show an aggregate number with the dropdown symbol (▼) next to it. Clicking would show a dropdown with the individual unread counts, to which you can click to open the individual tabs.

Thoughts?

**Comment by gfpacheco** [comment]
I don't think the dropdown would show up for everyone, only for those with more than one account. It could even show up only if there's unread email in both inboxes.

I liked the "priority" idea, I don't mind losing this kind of information, since I'm addicted to keep my inboxes clean, but this is me.

**Comment by elithrar** [comment]
> Another idea is to show an aggregate number with the dropdown symbol (▼) next to it. Clicking would show a dropdown with the individual unread counts, to which you can click to open the individual tabs.

I like this. Without getting too deep into "option-itis", an option to show either "primary account count" or "aggregate count" would be sufficient.

**Comment by mpeetoom** [comment]
Adding multiple accounts as in "Notifier for Gmail". 
![afbeelding 2017-07-20-080700](https://user-images.githubusercontent.com/30310910/28404704-b30116b4-6d2a-11e7-843b-34c47d6d2335.jpg)
![afbeelding 2017-07-20-080714](https://user-images.githubusercontent.com/30310910/28404714-ba283d6e-6d2a-11e7-8f7f-4665be6026d9.jpg)
